src/events/messageCreate.ts

```diff
@@ -36,7 +43,7 @@ const events: DiscordEvent<"messageCreate"> = {
   execute: async (message) => {
     // deny handling if author is bot or message is empty (new user join server)
     wrapError(message, async () => {
-      if (message.author.bot || !message.content) return
+      if (message.author.bot || (!message.content && !message.stickers)) return
```

Comment:
```suggestion
      if (message.author.bot || (!message.content && !message.stickers.size)) return
```
`message.stickers` là object `Collection` nên `!message.stickers` luôn false